src/Client.ts

```diff
@@ -130,15 +302,19 @@ export class Client {
             });
     }
 
-    private static extractCustomSessionIdFromLoginResponse(response): string {
-        const vehicleInfoList = response.VehicleInfoList;
-
-        if (!vehicleInfoList) {
-            console.warn('Response did not include a vehicle information list.');
-            return;
-        };
-
-        const vehicleInfo = vehicleInfoList.vehicleInfo;
+    private static extractCustomSessionIdFromLoginResponse(response, regionCode): string {
+        var vehicleInfo;
+        if (regionCode === "NNA") {
+            const vehicleInfoList = response.VehicleInfoList;
+            if (!vehicleInfoList) {
+                console.warn('Response did not include a vehicle information list.');
+                return;
+            };
+
+            vehicleInfo = vehicleInfoList.vehicleInfo;
+        } else { // NE region
+            vehicleInfo = response.vehicleInfo;
```

Comment:
I'm curious as to whether this is really just a quirk of the `NE` endpoint or if it's really just the "new" response across regions, and `NNA` is now broken.  As I've no longer access to a Leaf, I cannot test this.

Reply:
Yes that's a good question. I'm unable to test the NNA region to see which it is. Perhaps it would be safer to check if the extra VehicleInfoList attribute exists in the JSON and handle it cleanly if not?